### PR TITLE
[ticket/14289] Add events to navbar header

### PR DESCRIPTION
**Pull request by RMcGirr83**
Before notification area and after user drop down area

https://tracker.phpbb.com/browse/PHPBB3-14289

PHPBB3-14289

### CODE REVIEW

**Comment by marc1706** [comment]
I'm really unsure about the event name. It doesn't really fit the location it is at. What about something like profile, user area or something like that instead of navigation.

**Comment by RMcGirr83** [comment]
I am completely open to suggestions.  Only reason I chose navigation was it is placed in the navigation header.

**Comment by marc1706** [comment]
Well that file already contains events that are called _navgation_append and _navigation_prepend.

My suggestions were profile or user area.

**Comment by RMcGirr83** [comment]
Okay, how about 

`navbar_header_user_profile_prepend` and `navbar_header_user_profile_append`

**Comment by marc1706** [comment]
Sounds good to me.

**Comment by RMcGirr83** [comment]
RC-1 or ?

**Comment by Elsensee** [comment]
Please use 3.1.8-RC1 :)

**Comment by RMcGirr83** [comment]
Travis and @paul999 give me a headache

**Comment by Elsensee** [comment]
About Travis: I have restarted the two failing buils. I think the failure was unrelated to your change.

I can't help you with @paul999, though.

File: phpBB/docs/events.md [hunk 1]
```diff
@@ -1072,6 +1072,20 @@ navbar_header_logged_out_content
...
 * Purpose: Add text and HTML in place of the username when not logged in.
 
+navbar_header_user_profile_prepend
+===
+* Locations:
+    + styles/prosilver/template/navbar_header.html
+* Since: 3.1.8-RC1
+* Purpose: Add links to the left of the notification area
+
+navbar_header_user_profile_append
```
Comment (marc1706) [inline phpBB/docs/events.md]: _append needs to be before _prepend

File: phpBB/docs/events.md [hunk 2]
```diff
@@ -1072,6 +1072,20 @@ navbar_header_logged_out_content
 * Since: 3.1.0-RC1
 * Purpose: Add text and HTML in place of the username when not logged in.
 
+navbar_header_user_profile_append
```
Comment (marc1706) [inline phpBB/docs/events.md]: I'm sorry but the placement is still wrong. In alphabetic order, navbar_header_u... should be after navbar_header_p...

Comment (RMcGirr83) [inline phpBB/docs/events.md]: Actually it should be after `navbar_header_q` and before `navbar_header_username`?

Comment (marc1706) [inline phpBB/docs/events.md]: Yes, of course. I was just pointing out that this is incorrectly placed right now and should be placed in alphabetical order.

**Pull request merged**